10up/retro-winamp-block
# Prevent LockedStream console error by providing metaData in advance

### Description of the Change

This change resolves a reported error that occurs due to the metaData key being missing from the initialTracks provided to Webamp.

 You can reproduce this error on the official CodeSandbox demo by just removing the metaData key: https://codesandbox.io/s/y0xypox60z.


Closes #46 

### How to test the Change
1. Load the block on any page and confirm no LockedStream promise errors appear in the console and the tracks have the correct metadata.

### Changelog Entry
> Fixed - Updated initialTracks to include metaData to prevent a LockedStream console error.


### Credits
Props @iansvo @ajmaurya99 


### Checklist:
